.travis.yml

```diff
@@ -1,14 +1,19 @@
+sudo: false
```

Comment:
We should refrain from using the `sudo` tag as it is deprecated as per https://github.com/IBM/portieris/issues/56

What do we need this for?